client/src/containers/Header.js

```diff
@@ -344,43 +346,43 @@ class Header extends React.Component {
                     />
                   </div>
                 </div>
-
-                <div
-                  className={`single-dropdown --${
-                    this.props.web3 && "--hidden"
-                  }`}
-                >
-                  <p onClick={() => this.setActiveTab(2)}>
-                    <i className="fas fa-network-wired"></i>
-                    <span>{strings.Networks}</span>
-                  </p>
-                  <div className={this.getDDClassName(2)}>
-                    {Object.values(constants.NETWORKS_INGAME).map(
-                      (network, index) => {
-                        if (network && network.name !== "local") {
-                          if (Number(network.id) === this.state.chainId)
-                            return false; // filter out current network
-                          return (
-                            <div
-                              key={index}
-                              onClick={(e) => {
-                                e.preventDefault();
-                                this.changeNetwork(network);
-                              }}
-                              className="dropdown-pill"
-                            >
-                              <a id={network.name} key={network.name} href="/">
-                                {network.name}
-                              </a>
-                            </div>
-                          );
+                {this.props.web3 && !store.getState().gamedata.readOnly && (
```

Comment:
Once in read-only mode, it's not obvious how to exit read-only mode.  This could be addressed by doing one or more of the following if in read-only (but I don't think these are a must for this first iteration):
- Adding a button here that brings back the MetaMask modal (or even just directly connects)
- If the user clicks the "Play now!" button on the main page, this is an indication that the user actually wants to connect.  In this case, exit read-only mode (or bring back the MetaMask modal, or directly connect)

Reply:
I like both options! thanks for the suggestion!

Reply:
This works great now, thanks a lot for addressing these so quickly!